docs/maya/skin_merge.md

```diff
@@ -0,0 +1,78 @@
+# AdnSkinMerge
+
+AdnSkinMerge is a Maya deformer to blend animation and simulation together. It allows for the merging of several animation and simulation meshes into a single final mesh.
+
+The influence simulation or animation meshes will have on the final mesh can be freely painted and modified by painting a blend weights map.
+
+### How To Use
+
+AdnSkinMerge makes use of its own User Interface to create and modify the deformer.
+
+To create an AdnSkinMerge deformer within a Maya scene, the following inputs must be provided:
+
+  - **Final Mesh (F)**: Mesh to apply the merge results.
+  - **Animation Mesh List (Al)**: Mesh(es) to drive the simulation skin.
+  - **Simulation Mesh List (Sl)**: Mesh(es) with either an AdnSkin deformer applied or with the results from the skin simulation applied.
+
+The process to create an AdnSkinMerge deformer is:
+
+1. Press ![Skin merge button](images/adn_skin_merge.png){style="width:4%"} in the AdonisFX shelf or *Skin Merge* in the AdonisFX menu, under the *Deformers* submenu in the *Create* section to open the following UI.
+
+<figure markdown>
+  ![create skin merge UI](images/create_skin_merge.png) 
+  <figcaption><b>Figure 1</b>: Create Skin Merge UI.</figcaption>
+</figure>
+
+2. With the UI open select the final mesh in the scene and press the *Add Selected* button in the *Final Mesh* section.
+
+3. Add the animation and simulation meshes taking into consideration the following requirements:
+    - At least one mesh must be added in each field.
+    - To add meshes to any list, select the meshes in the scene and click the respecitve *Add Selected* button.
+    - Adding the same mesh twice to a list is not supported.
+    - Adding the same mesh as a Simulation Mesh and as an Animation Mesh is not adviced but it is supported.
+    - If you wish to remove a single element from the list, select it in the Skin Merge UI and press the Remove Selected button.
+    - You may also clear any list fully by pressing the respective Clear button.
+
+4. Set a custom name to the deformer and specify the initialization time.
+
+5. Press the *Create* button and a message box will notify you that AdnSkinMerge has been created properly. The final mesh will follow the animation mesh inputs by default.
+
+6. To modulate the influence of the simulation mesh inputs, use the maya paintable context and customize the blend weights map.
+
+Once we have an AdnSkinMerge deformer created, if we wish to modify its input meshes (animation mesh list, simulation mesh list or both) we may follow this process:
+
+1. Go to *Deformers > Skin Merge* in the AdonisFX menu, under the *Edit* section.
+2. The following UI will get displayed. Here you will see listed the current animation and simulation meshes the deformer has connected. From this UI you may freely add or remove from either list Note that at least one element must be present in each list to be able to apply the changes. 
```

Comment:
I think we need a blank line between items in a list for proper rendering.

Reply:
Done!